src/components/Login/LoginParts/LoginSupport.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { Grid, Link } from "@mui/material";
+
+export const LoginSupport: React.FC = () => {
```

Comment:
create LoginSupport folder and move component

Reply:
change file structure